service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisTemplateImages.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class DocmosisTemplateImages {
```

Comment:
Thoughts on extending this to cover all images in `uk/gov/hmcts/reform/fpl/enums/DocmosisImages.java`